### PR TITLE
Added Job Stage querying to greenhouse gem

### PR DESCRIPTION
**Pull request by Sleachga**
Add methods for querying ALL job stages

### CODE REVIEW

File: lib/greenhouse_io/api/client.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 require 'greenhouse_io/api/candidate_collection'
 require 'greenhouse_io/api/scheduled_interview_collection'
 require 'greenhouse_io/api/job_collection'
+require 'greenhouse_io/api/job_stages_collection'
```
Comment (etipton) [inline lib/greenhouse_io/api/client.rb:5]: can you switch this to `job_stage_collection` to be consistent w/ other naming conventions in this repo?

File: lib/greenhouse_io/api/job_stages.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'greenhouse_io/api/resource'
+
+module GreenhouseIo
+  class JobStages < Resource
```
Comment (etipton) [inline lib/greenhouse_io/api/job_stages.rb:6]: `JobStage`

**Review by etipton — CHANGES_REQUESTED** [review]
naming convention changes

File: lib/greenhouse_io/api/client.rb (hunk 1 above)
Comment (Sleachga) [reply lib/greenhouse_io/api/client.rb]: Fixed in 22c0e83f3a843bc2788acfaf16a8d70013e337e0

**Pull request merged**